components/buttons/AcceptBid.tsx

```diff
@@ -76,10 +76,8 @@ const AcceptBid: FC<Props> = ({
         bidId={bidId}
         collectionId={collectionId}
         tokenId={token?.token?.tokenId}
-        onClose={() => {
-          if (mutate) {
-            mutate()
-          }
+        onClose={(data, stepData, currentStep) => {
+          if (mutate && currentStep == 4) mutate()
```

Comment:
Can we use the AcceptBidStep in here? Just to make it clearer.

Reply:
Ah yeah, didn't even realize I could do that. Will update